politeiad/backendv2/backendv2.go

```diff
@@ -35,6 +35,10 @@ var (
 	// ErrPluginCmdInvalid is returned when a invalid plugin command is
 	// used.
 	ErrPluginCmdInvalid = errors.New("plugin command invalid")
+
+	// ErrDuplicatePayload is returned when a duplicate payload is sent to
+	// a plugin, where it tries to write data that already exists.
```

Comment:
It's not really obvious why a duplicate payload would not be allowed.  I would add an brief explination of why this is an issue, i.e. timestamps rely on the hash of the payload so duplicate payloads are not allowed since they cause collisions.